src/run.spec.js

```diff
@@ -0,0 +1,81 @@
+const {
+  fetchAndSaveTrips,
+  fetchAndSaveManualEntries,
+  fetchTripsMetadata
+} = require('./lib')
+const { saveAccountData } = require('./save')
+
+jest.mock('./trace-requests')
+jest.mock('./lib', () => ({
+  fetchTripsMetadata: jest.fn(),
+  fetchAndSaveTrips: jest.fn(),
+  fetchAndSaveManualEntries: jest.fn()
+}))
+jest.mock('./utils', () => ({
+  ...jest.requireActual('./utils'),
+  restartKonnector: jest.fn()
+}))
+jest.mock('./save', () => ({
+  saveAccountData: jest.fn()
+}))
+const { restartKonnector } = require('./utils')
+const { TRIPS_CHUNK_SIZE } = require('./const')
+
+process.env.COZY_URL = 'http://test.cozy'
+
+const run = require('./run')
+
+describe('timeout', () => {
+  const accountData = {
+    lastSavedTripDate: Date.now(),
+    lastSavedManualDate: Date.now()
+  }
+  it('should not restart execution when timeout is not reached', async () => {
+    process.env.COZY_TIME_LIMIT = 5000
```

Comment:
By reading run.js COZY_TIME_LIMIT seems to be in seconds, but reading 5000 here confuses me.

Reply:
It is indeed in seconds, but I agree the value confusing. I'll change it :)

Reply:
Done with d267955